addons/website_crm/views/website_templates_contactus.xml

```diff
@@ -2,13 +2,10 @@
 <odoo>
     <data>
         <template id="contactus_form" name="Contact Form (Opportunity)" inherit_id="website.contactus">
-            <xpath expr="//span[@data-for='contactus_form']" position="before">
+            <xpath expr="//t[@t-set='contactus_form_values']" position="after">
                 <t t-set="contactus_form_values" t-value="dict(contactus_form_values, **{
                     'contact_name': request.params.get('contact_name', ''),
-                    'phone': request.params.get('phone', ''),
```

Comment:
I would have kept those but it's fine I guess, a change in website could now break `website_crm`, eg if we remove the phone input from the default form.

Reply:
That's true for every single line of code which is in website, don't know why these ones should be duplicated :wink: